test/e2e/gce/gce_test.go

```diff
@@ -0,0 +1,78 @@
+package gce
```

Comment:
The main reason that we will have yet another testing flagset,pkg to maintain, which i want to avoid if possible.